pkgs/games/crossfire/default.nix

```diff
@@ -0,0 +1,38 @@
+pkgs:
+
+let
+  callPackage = pkgs.callPackage;
+in rec {
+  crossfire-client = callPackage ./crossfire-client.nix {
+    version = "1.73.0"; rev = 20680;
+    sha256 = "192y9nmskrx907h16p8rl4cvksnh5m0q8kz9cxw376sq7gam7g0y";
+  };
```

Comment:
Can we limit the number of versions here?

Reply:
How do you mean? Ditch the latest/stable separation? Enough has changed (with no official release) since 1.7.1 that I think it does make sense to have both available -- there are still 171 servers out there.